build.gradle

```diff
@@ -11,8 +11,8 @@ buildscript {
         gradlePluginPortal()
     }
     dependencies {
-        classpath group: 'kr.motd.gradle', name: 'sphinx-gradle-plugin', version: '1.0.3.Final' // can be upgraded to 2.2.0
-        classpath "jp.classmethod.aws:gradle-aws-plugin:0.35"  // can be upgraded to 0.37
+        classpath group: 'kr.motd.gradle', name: 'sphinx-gradle-plugin', version: '1.0.3.Final'
+        classpath "jp.classmethod.aws:gradle-aws-plugin:0.41"
```

Comment:
We should use the same version across build gradles - 78229ee

`classpath "jp.classmethod.aws.reboot:gradle-aws-plugin-reboot:0.42"`

Sorry for the late reply on this one, also